### PR TITLE
chore: add helm

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: helm/templates/mapproxy-routes.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+kind: Route
+metadata:
+  name: {{ template "map-proxy.fullname" . }}-map-proxy-route
+  annotations:
+    haproxy.router.openshift.io/rewrite-target: /
+spec:
+  path:  {{ .Values.route.mapProxyPath }}
+  to:
+    kind: Service
+    name: {{ template "map-proxy.fullname" . }}-nginx
```
Comment (asafMasa) [inline helm/templates/mapproxy-routes.yaml:12]: nginx?

**Review by asafMasa — APPROVED** [review]

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# ====================================================================================================
+# ====================================================================================================
+# ==================================Global Configurations=============================================
+# ====================================================================================================
+# ====================================================================================================
+pullSecretName: acr-registry
```
Comment (Yossizz) [inline helm/values.yaml:6]: please remove

File: helm/values.yaml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+imagePullPolicy: Always
+caLocation: null
+caFileName: null
+caSecretName: null
+opensslName: null
+openSSL: null
+splunkIndex: ""
+
+global:
+  dnshost: apps.v0h0bdx6.eastus.aroapp.io
```
Comment (Yossizz) [inline helm/values.yaml:16]: please remove

File: helm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    SSL_ENABLED: false
+    FORCE_PATH_STYLE: true
+
+# ====================================================================================================
+# Variables for map proxy
+# ====================================================================================================
+
+mapProxy:
+  enabled: true
+  mapProxyImage: "acrarolibotnonprod.azurecr.io/mapproxy"
```
Comment (Yossizz) [inline helm/values.yaml:119]: please remove

**Review by Yossizz — CHANGES_REQUESTED** [review]
please remove values

File: helm/values.yaml [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+caSecretName: null
+opensslName: null
+openSSL: null
+splunkIndex: ""
+
+global:
+  dnshost: apps.v0h0bdx6.eastus.aroapp.io
+  tracing: {}
+
+environment: development
```
Comment (CL-SHLOMIKONCHA) [inline helm/values.yaml:19]: adapt to the newest helm template and add it to the helper.tpl file, so we can use as global config.

File: helm/values.yaml [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+# cloudProvider is provider where you are going to deploy map exporter, it could be: aws, minikube
+cloudProvider: azure
```
Comment (CL-SHLOMIKONCHA) [inline helm/values.yaml:21]: adapt to the newest helm template and add it to the helper.tpl file, so we can use as global config.

File: helm/values.yaml [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+shared:
+  MAPPROXY_CONFIG_PROVIDER: db
+  ssl: # external paths
+    CA_DIR: /opt/certs/ca/
+    KEY_DIR: /opt/certs/key/
+    CERT_DIR: /opt/certs/cert/
+    CA_FILE_LOCATION: /opt/certs/ca/ca.pem
+    KEY_FILE_LOCATION: /opt/certs/key/key.pem
+    CERT_FILE_LOCATION: /opt/certs/cert/cert.pem
+  S3:
```
Comment (CL-SHLOMIKONCHA) [inline helm/values.yaml:107]: s3 configs should be taken from global config as well

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]
first, in my opinion its wrong to separate the mapproxy helm from init-container and mapproxinator (how did you checked it with no init-container?)
second,  its not updated to the last helm template that we are using with the common global config.

**Pull request closed without merging**